lib/jekyll/configuration.rb

```diff
@@ -11,7 +11,8 @@ class Configuration < Hash
       'destination'   => File.join(Dir.pwd, '_site'),
       'plugins'       => '_plugins',
       'layouts'       => '_layouts',
-      'data_source'   =>  '_data',
+      'data_source'   => '_data',
+      'includes_dir'  => '_includes',
```

Comment:
@parkr turns out I'd also need to update `jekyll-sass-converter` if we rename `source` to `source_dir`. Is that still something you would want?